applications/nrf5340_audio/unicast_server/main.c

```diff
@@ -410,10 +411,12 @@ static int zbus_link_producers_observers(void)
 		return ret;
 	}
 
-	ret = zbus_chan_add_obs(&volume_chan, &volume_evt_sub, ZBUS_ADD_OBS_TIMEOUT_MS);
-	if (ret) {
-		LOG_ERR("Failed to add volume sub");
-		return ret;
+	if (IS_ENABLED(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP)) {
```

Comment:
This should be the same on the standard DK?

Reply:
I believe the volume on the server is controlled via the hardware codec, which isn't on the standard DK?

Reply:
My mistake. You are right, but I am unsure if we should filter that here, or on the zbus subscriber side.

Reply:
This approach should result in less memory usage and cycles.